src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/AlignedContig.java

```diff
@@ -24,12 +24,15 @@
     public final String contigName;
     public final byte[] contigSequence;
     public final List<AlignmentInterval> alignmentIntervals;
+    public final boolean hasEquallyGoodAlnConfigurations;
 
-    public AlignedContig(final String contigName, final byte[] contigSequence, final List<AlignmentInterval> alignmentIntervals) {
+    public AlignedContig(final String contigName, final byte[] contigSequence, final List<AlignmentInterval> alignmentIntervals,
+                         final boolean hasEquallyGoodAlnConfigurations) {
```

Comment:
Are we ever setting this to anything other than `false`? I don't see anywhere in the code where it's set to `true` but I may have missed it. At any rate, it's not clear to me how this is set or what exactly the field is supposed to mean.